guides/source/action_view_overview.md

```diff
@@ -913,7 +913,7 @@ hidden_field(:user, :token)
 
 #### label
 
-Returns a label tag tailored for labelling an input field for a specified attribute.
+Returns a label tag tailored for labeling an input field for a specified attribute.
```

Comment:
Unless you are **really** sure that they American English across their whole Rails guide.